kvconfig.yml

```diff
@@ -0,0 +1,11 @@
+routes:
+  unknown-error-alerts:
+    matchers:
+      title: ["stats"]
+    output:
+      type: "alerts"
+      series: "kinesis-consumer.notifications.unknown-error"
+      dimensions: [ ]
```

Comment:
`kinesis-consumer.notifications.unknown-error`

I think it would be preferable to send to a series called `kinesis-consumer.unknown-error` and pass a dimension which contains the consumer type (e.g. we could use `source`, which already contains the type of consumer)

This way all this related data is contained in a single series name, instead of a separate one per app

Reply:
(same comment relates to the other PRs where we've made the same change)

Reply:
this is because source is set to `kinesis-consumer`